src/bauhaus/bauhaus.c

```diff
@@ -2205,9 +2205,7 @@ static gboolean dt_bauhaus_slider_scroll(GtkWidget *widget, GdkEventScroll *even
   int delta_y;
   if(dt_gui_get_scroll_unit_delta(event, &delta_y))
   {
-    if(delta_y == 0) return TRUE;
-    gdouble delta;
-    delta = delta_y * -w->data.slider.scale / 5.0;
+    gdouble delta = delta_y * -w->data.slider.scale / 5.0;
```

Comment:
Given the emulated events remark I just posted, you may want to keep the conditional return and skip the next two statements in duplicate calls to avoid wasting time with `dt_bauhaus_slider_add_delta_internal` specifically.

Reply:
I removed the conditional return because `if(dt_gui_get_scroll_unit_delta(event, &delta_y))` evaluates to true only when `delta_y != 0`. Is there some edge case that I didn't notice?

Reply:
Ah, I got confused because I was looking at the individual commit which was reverting a test that you had added yourself earlier.

I think (but haven't tested) that you could get a zero result if there are offsetting x and y moves that are added together. By scrolling diagonally on the touchpad. But that would be just a minor corner case.

Reply:
You are right. It is possible to get zero and that actually causes the slider to move out of range in some cases.

Reply:
Does it? It's still a relative move, so a zero move shouldn't change anything, just use up time unnecessarily? But I have seen sliders jump, so if this _were_ the cause, and solved, that would be great.

Reply:
Looking closer it actually causes division by zero in `dt_bauhaus_slider_add_delta_internal`. Solves the issue for me.

Reply:
Argh!! Would you mind fixing it in dt_bauhaus_slider_add_delta_internal as well (or instead) for me please?

Just a check for delta == 0 -> return on the first line should do it...?

Reply:
Added the check.